doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1051 @@
+* Enabled Thread certification by inheritance.
+* HomeKit samples now use qualified Bluetooth LE libraries.
+
+nRF IEEE 802.15.4 radio driver
+------------------------------
+
+* Production support for nRF5340 in single protocol configuration.
+* Development support for nRF5340 in multiprotocol configuration (IEEE 802.15.4 and Bluetooth LE).
+* Added PA/LNA GPIO interface support for RF front-end modules (FEM) in the radio driver. The front-end module feature in the radio driver currently has support for the SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+
+Thread
+------
+
+* Added:
+
+  * Production support for nRF5340 in single protocol configuration.
+  * Development support for nRF5340 in multiprotocol configuration (Thread and Bluetooth LE).
+  * Support for nRF5340 for all samples except the :ref:`ot_ncp_sample` sample.
+  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Thread. The front-end module feature in Thread currently has support for SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+
+* Updated:
+
+  * Optimized ROM and RAM used by Thread samples.
+  * Disabled Hardware Flow Control on the serial port in :ref:`coap_client_sample` and :ref:`coap_server_sample` samples.
+  * Thread 1.1 pre-built libraries:
+
+    * Included the latest OpenThread changes.
+    * Added libraries for nRF5340 platform.
+    * Libraries will be certified after the release in multiple feature variants (certification IDs and status can be found in the compatibility matrices in the `Nordic Semiconductor Infocenter`_).
+
+Zigbee
+------
+
+* Added:
+
+  * Development support for :ref:`NCP (Network Co-Processor) <ug_zigbee_platform_design_ncp_details>`.
+  * Development support for the nRF5340 DK in single and multi-protocol (Zigbee and Bluetooth LE) configuration for the :ref:`zigbee_light_switch_sample`, :ref:`zigbee_light_bulb_sample`, and :ref:`zigbee_network_coordinator_sample` samples.
+  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Zigbee. The front-end module feature in Zigbee currently has support for SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+  * :ref:`zigbee_ncp_sample` sample, which is a Network Co-Processor example for nRF52840 (DK and dongle) and nRF52833 DK.
+
+	* Supports USB and UART transports.
+	* Enables USB DFU when USB transport is used.
+
+  * New ``zcl ping`` command in the :ref:`lib_zigbee_shell` library.
+  * New libraries that were extracted from common code under :file:`subsys/zigbee/common`:
+
+    * :ref:`lib_zigbee_application_utilities` library
+    * :ref:`lib_zigbee_logger_endpoint` library
+
+* Updated:
+
+  * ZBOSS Zigbee stack to version 3_3_0_6+11_30_2020. See :ref:`zboss_configuration` for detailed information.
+  * Added development (unstable) :ref:`zboss` libraries (v3.4.0.800). See :ref:`zboss_configuration` for detailed information.
```

Comment:
Incorrect version: should be: `3.5.0.0`, as in nrfxlib change log.

Reply:
Updated